jqjq.jq

```diff
@@ -2612,13 +2652,14 @@ def jqjq($args; $env):
         elif $opts.slurp then [inputs]
         else inputs
         end;
-      ( if $opts.no_builtins then {}
+      parse_colors($opts; $env) as $colors
+    | ( if $opts.no_builtins then {}
         else builtins_env
         end
       ) as $builtins_env
     | _inputs
     | eval($opts.filter; {"$ENV": $env}; $builtins_env)
-    | tojson
+    | _tojson({$colors})
     );
 
   ( ( { filter: "."
```

Comment:
Wonder it would be nice to have a  `_parse_env` that parses colors etc and combine into `$opts` that can be passed to `_filter` and `_repl`? then the default colors config could be in this object?

What do you think?

Reply:
Good idea. I've done that now in https://github.com/wader/jqjq/pull/14/commits/39bd5edb2c4bab52ef6002676b4710c3ddd59771. I called it `_parse_opts` because it parses both the command line options and environment, and the result is `$opts`. I think “env” is ambiguous, because jqjq usually uses it to mean bindings in jq.